src/scss/global.scss

```diff
@@ -385,21 +385,25 @@ h4 {
   }
 
   .left-column {
+    box-shadow: 0 0 30px $black;
     transition: all 0.1s linear;
     width: 70px;
     display: flex;
     flex-direction: column;
     align-items: center;
     padding: 10px;
     background: $dark;
-    border-radius: 22px;
+    border-radius: 0 22px 22px 0;
     cursor: pointer;
     overflow: auto;
-    height: 100%;
+    position: absolute;
+    top: 40px;
+    z-index: 1;
   }
 
   .left-column.expanded {
     width: 300px;
+    height: 450px;
```

Comment:
This should be allowed to grow based on the content size

Reply:
the div is scrollable. On the mock-ups, it doesn't take full height. Full height doesn't look good at all in my opinion